### PR TITLE
[hotfix][docs] Readme review to clarify heading formatting

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
I clarified how to format a heading casing wise.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ This will be replaced with the value of the variable called `NAME` when generati
 
 #### Headings
 
-All documents are structured with headings. From these headings, you can automatically generate a page table of contents (see below).
+All documents are structured with headings, written in "Title Case". From these headings, you can automatically generate a page table of contents (see below).
```
Comment (greghogan) [inline docs/README.md]: It seems that this could be left as an implicit assumption for "headings".

File: docs/index.md [hunk 2]
```diff
@@ -23,19 +24,17 @@ specific language governing permissions and limitations
...
-
-
 This documentation is for Apache Flink version {{ site.version_title }}. These pages were built at: {% build_time %}.
 
 Apache Flink is an open source platform for distributed stream and batch data processing. Flink’s core is a streaming dataflow engine that provides data distribution, communication, and fault tolerance for distributed computations over data streams. Flink builds batch processing on top of the streaming engine, overlaying native iteration support, managed memory, and program optimization.
 
 ## First Steps
 
-- **Concepts**: Start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you understand other parts of the documentation, including the setup and programming guides. We recommended you read these sections first.
+-   **Concepts**: We recommend you start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you understand other parts of the documentation, including the setup and programming guides.
```
Comment (greghogan) [inline docs/index.md]: Why the extra spacing?

File: docs/index.md [hunk 3]
```diff
@@ -23,19 +24,17 @@ specific language governing permissions and limitations
...
 
-- **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [study some examples](examples/index.html).
+-   **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [study some examples](examples/index.html).
 
-- **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html) and the [DataStream API](dev/datastream_api.html) or the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
+-   **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html), the [DataStream API](dev/datastream_api.html) or the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
```
Comment (greghogan) [inline docs/index.md]: Oxford comma?

File: docs/README.md (hunk 1 above)
Comment (ChrisChinchilla) [reply docs/README.md]: @greghogan Not sure, I have found a lot of inconsistent headings in the docs, so it seemed worthwhile adding, doesn't mean anyone will follow it of course :)

File: docs/index.md (hunk 2 above)
Comment (ChrisChinchilla) [reply docs/index.md]: @greghogan It's something my Markdown linter 'fixes', it doesn't matter too much, but is apparently more correct.

File: docs/index.md (hunk 3 above)
Comment (ChrisChinchilla) [reply docs/index.md]: @greghogan No, this was an attempt to make the whole passage flow better as it says…

> xxx and yyy or zzz

So I was attempting to seperate out the two parts of the paragraph, better may be…

> **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html), the [DataStream API](dev/datastream_api.html), and the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.

Then it's an oxford comma, but that's stylistic.

**Comment by aljoscha** [comment]
I'm closing this as "Abandoned", since there is no more activity and the code base has moved on quite a bit. Please re-open this if you feel otherwise and work should continue.

**Pull request closed without merging**